### ISSUE TITLE
Move the Percentage sub-dialog into the Summaries sub-dialog in Prepare > Data Reshape > Column Summaries

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#8729]
The calculation system in R-Instat is one of our "selling points".  It needs a presentation (including video) and to be well reported in the help.  As preparation this is the first of 3 urgent(?) issues we need to undertake.

I need an initial discussion with @volloholic before this work.  I'll report once that's done.

This one is to get rid of the Propn and Percentages button and sub-dialog from this dialog.  Instead it adds a tab to the summaries sub-dialog. 

![image](https://github.com/africanmathsinitiative/R-Instat/assets/11226469/05662687-0c78-4b84-9b4e-d49a4e8ad896)

Instead - and I hope this will be easy to do - it is just called `Percentages`, and becomes a further tab on the Summaries sub-dialog.

I think this (with small changes) just needs to be transferred (and corrected) to become the last tab in the existing Summaries sub-dialog. 

Here it is: 

![image](https://github.com/africanmathsinitiative/R-Instat/assets/11226469/2cbc45ca-03da-486f-87af-368e6d8645e1)

a) We still need the checkbox at the top, simplify the label to become `Calculate Percentages`.
b) Then, either we implement the other radio buttons at the top, or let's get rid of them.
c) Check, notice this is a different data frame here to the main dialog.  I think this should not be allowed, so the dataframe part of the data selector is disabled.
d) Correct the R code.  I think it works ok when factors are included, but it should also work when they are omitted?  I assume, with this example it should add the percentage of values for each variety.  Currently it doesn't.  (This may need a quick input from @lilyclements, but I suggest that could come after the transfer has been completed.

**Comment by lilyclements** [comment]
> Instead - and I hope this will be easy to do - it is just called Percentages, and becomes a further tab on the Summaries sub-dialog.

@rdstern I agree with this change.

>  I think it works ok when factors are included, but it should also work when they are omitted? I assume, with this example it should add the percentage of values for each variety. 

By this, do you mean that you would expect the percentage of values for each variety when the variable "Variety" is in the "Summarise by" receiver, but not in the sdg multiple receiver?

If we had multiple factors in our "Summarise by" receiver, but wanted only row percentages and not column percentages, we wouldn't want to do the percentage by all variables in our "Summarise by" receiver. Or am I misunderstanding your point?
Could the multiple receiver on the sub dialog contain the variables in the "Summarise by" receiver by default, but can be removable, or is that too complex for the user?

**Comment by rdstern** [comment]
@lilyclements thanks for the comments.  I have been trying the existing button a little and have the impression that it is working ok, and is already pretty powerful.  I think that includes all your column and row percentages for frequencies and also percentages of anything else, e.g. percentage change in mean rainfall, where there is quite a lot of choice in how we define the "mean rainfall" as we summarise.

There may be more to be done, but my hope is first to leave the sub-dialog (with its current R-code) exactly as it is.  So we just move it into the Summarise sub-dialog.  That makes the Prepare > Data Reshape > Column Summaries main dialog a bit simpler, because we lose the second button.  Of course the sub-dialog is a bit more complicated.

But the big improvement is then that the single sub-dialog is then also immediately there for the now single sub-dialog that is also called by the Tables, (and the one-variable and 2/3 Variables Summarise) dialogs in the describe menu.  That's a big change.

Once it is moved I would like to return to some of your questions above, to see if mmore is needed in addition to sorting out the documentation.

**Comment by lilyclements** [comment]
@rdstern sounds like a good plan - and hopefully shouldn't be difficult to shift it over. Who do you suggest takes this on?

**Comment by rdstern** [comment]
@lilyclements agreed and I hope to tell you when we discuss this afternoon.  I've sent a skype request to @N-thony to discuss this, but he has yet to answer.

**Comment by rdstern** [comment]
@N-thony I reflected following our discussion.  To make it really easy I propose as follows on the More tab of the current sub-dialog.:

![image](https://github.com/IDEMSInternational/R-Instat/assets/11226469/2d3c88e4-19fa-492b-a98a-aab67c9b6f1a)

a) For now we simplym make the Kurtosis box smaller - perhaps 1/3 its current size.  T
hen add another with label Percentages.  It has a checkbox, default unchecked, with label `Add Percentages`  There is also a button with label `Options`.  If that is pressed, then it opens the current Percentages sub-dialog.  It is exactly as now, with the checkbox checked, (or omitted).
